animate/adapt.py

```diff
@@ -41,11 +56,16 @@ class MetricBasedAdaptor(AdaptorBase):
     """
 
     @PETSc.Log.EventDecorator()
-    def __init__(self, mesh, metric, name=None):
+    def __init__(self, mesh, metric, name=None, comm=COMM_WORLD):
```

Comment:
*** Marking all connected comments with these stars ***

`comm=None`

Reply:
Why? `None` is not a valid communicator.

Reply:
Fixed in 83d8b81.